sequencing-server/src/lib/codegen/SeqJsonSpec.ts

```diff
@@ -0,0 +1,309 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type VariableDeclaration = {
```

Comment:
Can we import this from the library directly instead of copying the file here? Added some docs on how to do this: https://github.com/NASA-AMMOS/seq-json-schema#typescript-types

Reply:
@camargo The types.ts didn't install when running `npm install`. 

These were the only files in `node_modules`:

LICENSE
package.json
README.md
schema.json

Reply:
Ah yes it would help if I actually publish the types 😃 
Try [1.0.9](https://www.npmjs.com/package/@nasa-jpl/seq-json-schema/v/1.0.9), it should have the `types.ts` file now.

Reply:
Thanks that worked!